lib/hooks.js

```diff
@@ -33,7 +33,15 @@ const beforeRunHook = ({ emitCountMap, manifestFileName }, compiler, callback) =
 };
 
 const emitHook = function emit(
-  { compiler, emitCountMap, manifestAssetId, manifestFileName, moduleAssets, options },
+  {
+    compiler,
+    emitCountMap,
+    manifestAssetId,
+    manifestFileName,
+    moduleAssets,
+    assetTypeModuleAssets,
+    options
+  },
```

Comment:
A few of these cs changes are being made automatically by the linter - I tried to keep the diff as clean as possible :)